DEVELOPING.md

```diff
@@ -1,63 +1,64 @@
-    init: true
-    ipc: host
-    ports:
-      - '5000:5000'
-    command:
-      [
-        "/bin/sh",
-        -c,
-        "cd /home/pwuser && npx -y playwright@<version> run-server --port 5000"
-      ]
-    network_mode: host
-```
-Example compose.yaml using playwright version 1.46.0 and port 5000
-
-```
-services:
-  playwright:
-    image: mcr.microsoft.com/playwright:v1.46.0-jammy
-    restart: always
-    container_name: "playwright"
-    init: true
-    ipc: host
-    ports:
-      - '5000:5000'
-    command:
-      [
-        "/bin/sh",
-        -c,
-        "cd /home/pwuser && npx -y playwright@1.46.0 run-server --port 5000"
-      ]
-    network_mode: host
-```
-Run the compose.yaml file using podman-compose command
-``` 
-podman-compose -f compose.yaml up
-```
-The output for the above command following the container in Ready state would be,
+
+Playwright is officially [supported](https://playwright.dev/docs/intro#system-requirements)
+to:
+- Windows 10+, Windows Server 2016+ or Windows Subsystem for Linux (WSL)
+- macOS 13 Ventura, or later
+- Debian 12, Ubuntu 22.04, Ubuntu 24.04, on x86-64 and arm64 architecture
+
+To run Playwright on [unsupported Linux distributions](https://github.com/microsoft/playwright/issues/26482)
+like Fedora, Playwright can be configured on docker/podman and the tests can be
+executed from the client (local machine). To do this, follow the section below.
+
+## Running Playwright as Docker/Podman Container
+
+First, clean-install the requirements:
+```sh
+npm ci
 ```
-Listening on ws://127.0.0.1:5000/
+
+Then, get the Playwright version (it is important that client and server
+versions of Playwright must match, otherwise you get `<ws unexpected response>
+ws://localhost:5000/ 428 Precondition Required`-like error when you try to run
+tests):
+```sh
+export PLAYWRIGHT_VERSION="v$(npx playwright --version | cut -d' ' -f2)"
+```
+
+By default, Playwright is listening on port `5000` (the default value of
+`PLAYWRIGHT_PORT` from `.devcontainer/.env`). You can override this value if it
+is already taken by the system or other application:
+```sh
+export PLAYWRIGHT_PORT=<your choice of port number>
 ```
 
-Now the user can execute the Playwright tests using the below command
+Then, start the Playwright service (you can override `.devcontainer/.env` by
+exporting environment variables with your own values as demonstrated above or
+you can just pass to `{docker,podman}-compose` your own `.env` file via
+`--env-file <env_file>` CLI option):
+```sh
+podman-compose -f .devcontainer/compose.yaml up
 ```
+
+After a while, the container should be in `Ready` state and you should see the
+output (replace `5000` by the value of `PLAYWRIGHT_PORT`):
+```
+Listening on ws://127.0.0.1:5000/
+```
+
+Now you can execute the Playwright tests (again, replace `5000` by the value of
+`PLAYWRIGHT_PORT`):
+```sh
 TRUSTIFY_URL=http://localhost:8080 PW_TEST_CONNECT_WS_ENDPOINT=ws://localhost:5000/ npx playwright test
```

Comment:
About this command:
```sh
TRUSTIFY_URL=http://localhost:8080 PW_TEST_CONNECT_WS_ENDPOINT=ws://localhost:5000/ npx playwright test
```
I know that command was not setup in this PR but since we are using `playwright-bdd` this command should be updated to 

```sh
TRUSTIFY_URL=http://localhost:8080 PW_TEST_CONNECT_WS_ENDPOINT=ws://localhost:5000/ npm run test
```

Could we include this change here please? Sorry for the extra work.

The `test` script within the package.json contains more than just `npx playwright test`.

Reply:
Done.